docs/admins.rst

```diff
@@ -51,6 +51,21 @@ Specifies the cleanup strategy for OSBS builds. By default this is set to
 ``true``. When it is set to ``true`` it'll remove the build pipeline on a
 completed or failed build.
 
+default_buildtime_limit
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Specifies default build time limit. If users don't specify a build time
+limit, this limit is used. It is integer set in seconds. If this value
+is not specified, the default value set by maintainers is used.
```

Comment:
Maybe I am misunderstanding, but the values default_buildtime_limit and max_buildtime_limit have to be always set by admins and not users right? If that's the case then the following line does not make sense to me:

> If this value is not specified, the default value set by maintainers is used.

Reply:
I am referring to `buildtime_limit` that can be set in `container.yaml`

Reply:
I see now. I was just a bit confused because in sentence "this limit is used" *this* refers to default_buildtime_limit but in "If this value is not specified" *this* refers to buildtime_limit.

Reply:
Ideally, if would like if this was a bit clearer (for people like me:D) but I acked this.